stats/views.py

```diff
@@ -15,8 +15,11 @@ def index(request):
     highscore = models.Profile.objects \
         .filter(expense__reimbursement__isnull=False, expense__expensepart__amount__lt=20000) \
         .annotate(total_amount=Sum('expense__expensepart__amount')) \
-        .filter(total_amount__gte=0) \
-        .order_by('-total_amount')[:10]
+        .annotate(receipts=Count('expense__expensepart')) \
+        .filter(total_amount__gte=0)
+
+    highscore_amount = highscore.order_by('-total_amount')[:10]
+    highscore_receipts = highscore.order_by('-receipts')[:10]
```

Comment:
I'm not sure why but it seems like one of my review comments didn't show up. 🤷 Or I'm just not seeing it.

I suggested a secondary ordering, like:
```python
highscore_receipts = highscore.order_by('-receipts', '-total_amount')[:10]
```

That would resolve ties in receipts by highest total amount.